README.md

```diff
@@ -5,3 +5,4 @@ The account/rank pairs which the Technical Committee should introduce.
 | Github Username | Kusama Account ID | Rank |
 | --- | --- | --- |
 | `gavofyork` | `FcxNWVy5RESDsErjwyZmPCW6Z8Y3fbfLzmou34YZTrbcraL` | 7 |
+| `jacogr` | `J9nD3s7zssCX7bion1xctAF6xcVexcpy2uwy4jTm9JL8yuK` | 4 |
```

Comment:
```suggestion
| `jacogr` | `J9nD3s7zssCX7bion1xctAF6xcVexcpy2uwy4jTm9JL8yuK` | 5 |
```

Update to V based on feedback from applicants, my initial mis-reading of IV meaning "> 4 years" and relative nature of the applications.